src/components/biggive-main-menu/biggive-main-menu.tsx

```diff
@@ -0,0 +1,121 @@
+import { Component, Element, Host, h } from '@stencil/core';
+
+@Component({
+  tag: 'biggive-main-menu',
+  styleUrl: 'biggive-main-menu.scss',
+  shadow: true,
+})
+export class BiggiveMainMenu {
+  @Element() host: HTMLBiggiveHeaderElement;
+  openMobileMenu = () => {
+    const mobileMenu = this.host.shadowRoot.querySelector('.nav-links') as HTMLElement;
+    mobileMenu.style.left = '0';
+  };
+  closeMobileMenu = () => {
```

Comment:
I think we probably need to expose something analogous to the header's `@Method`-annotated `closeMenuFromOutside()`, so that e.g. Angular app navigations can close the menu appropriately. Or is there another way you're planning to get the same result?

Reply:
Good shout.